config/config.go

```diff
@@ -41,6 +41,7 @@ const (
 	EnvServerMetricsPort                = "SERVER_METRICS_PORT"
 	EnvServerMetricsEnabled             = "SERVER_METRICS_ENABLED"
 
+	EnvGrafanaEnabled                  = "GRAFANA_ENABLED"
```

Comment:
The `EnvGrafanaDisplayLink` already does pretty much the same, so I think either don't add the new one or remove the old one.

Reply:
Removed. Thanks @jotak